graph/graph.py

```diff
@@ -23,13 +24,57 @@ class KytosGraph:
 
     def __init__(self):
         self.graph = nx.Graph()
+        self._accepted_metadata = {
+            'ownership',
+            'bandwidth',
+            'reliability',
+            'priority',
+            'utilization',
+            'delay',
+        }
         self._filter_functions = {
-            "ownership": lazy_filter(str, filter_in("ownership")),
-            "bandwidth": lazy_filter((int, float), filter_ge("bandwidth")),
-            "reliability": lazy_filter((int, float), filter_ge("reliability")),
-            "priority": lazy_filter((int, float), filter_le("priority")),
-            "utilization": lazy_filter((int, float), filter_le("utilization")),
-            "delay": lazy_filter((int, float), filter_le("delay")),
+            "ownership": EdgeFilter(
+                operator.and_,
+                lambda edge, val: frozenset(edge[2].get('ownership', {}).keys()) or val,
+                TypeCheckPreprocessor(
+                    str,
+                    lambda val: frozenset(val.split(','))
+                )
+            ),
+            "not_ownership": EdgeFilter(
+                lambda a, b: not (a & b),
```

Comment:
Same as my previous question, why not just providing a `operator.not_(operator.contains(*, *))` equivalent of `not in` filter? I think it'd be more readable and easier to understand, and then in the future when multiple ownership needs to be filtered as an input then we can get back to this discussion, unless you've already sorted that out with network engineers and documented it. Let me know if you had other plans/ideas in mind.

Reply:
Did it this way because this is equivalent to the logic that @italovalcy used [here](https://github.com/kytos-ng/pathfinder/issues/63#issuecomment-1819592771).

Reply:
I see. Makes sense, and it's also following a similar behavior with `undesired_links` in terms of filtering out any non empty intersection, so it's aligned with the existing multiple values exclusion filters.